src/opnsense/mvc/app/controllers/OPNsense/Base/ApiMutableTableModelControllerBase.php

```diff
@@ -77,10 +77,10 @@ public function getItemAction($uuid = null)
      * setItemAction is called. This hook is called after a model has been
      * constructed and validated but before it serialized to the configuration
      * and written to disk
-     * @param $mdl The validated model containing the new state of the model
+     * @param $uuid item unique id
      * @return Error message on error, or null/void on success
      */
-    protected function setItemActionHook($mdl)
```

Comment:
No longer pass the model in as we decided before it can just be got through $this->getModel(). Do however pass the uuid in because there's no easy way to get it.